serverless/jobs/how-to/reference-secret-in-job.mdx

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: How to reference secrets in Serverless Jobs
+  description: Steps to reference secrets from Secret Manager in your Serverless Jobs.
+content:
+  h1:  How to reference secrets in Serverless Jobs
+  paragraph: Steps to reference secrets from Secret Manager in your Serverless Jobs.
+tags: serverless jobs secrets secret-manager reference environment variable
+dates:
+  validation: 2024-10-27
+  posted: 2024-10-27
+categories:
+  - serverless
+  - jobs
+---
+
+Serverless Jobs seamlessly integrates with [Secret Manager](/identity-and-access-management/secret-manager/), which allows you to store, manage, and access sensitive information, such as credentials, SSH keys, SSL/TLS certificates, or any key/value pairs you need to secure.
+
+You can reference any secret stored in Secret Manager in a job, without having to hardcode any sensible data.
```

Comment:
I think you can add somewhere in the doc that "each access to a Secret by a Job Run generates a call to the Secret Manager API, refer to [Secret Manager pricing](https://www.scaleway.com/en/pricing/security-and-account/) for more information"